elastic/apm-server
# Update to elastic/beats@38fc1ed06525

## Motivation/summary

Update beats to fix system tests and bring in the custom kibana headers change.

## Checklist


I have considered changes for:
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
~- [ ] telemetry~

## How to test these changes

- Configure apm-server.kibana to connect to netcat or similar, so headers can be observed
- Configure `apm-server.kibana.headers='{"foo": "bar"}'``
- Run apm-server, check for the custom header

## Related issues

Closes https://github.com/elastic/apm-server/issues/4065
